ca_municipalities/people.py

```diff
@@ -0,0 +1,115 @@
+from utils import CSVScraper, CanadianPerson as Person
+from pupa.scrape import Organization, Post
+from collections import defaultdict
+
+import re
+
+
+class CanadaMunicipalitiesPersonScraper(CSVScraper):
+    csv_url = 'https://docs.google.com/spreadsheets/d/e/2PACX-1vRrGXQy8qk16OhuTjlccoGB4jL5e8X1CEqRbg896ufLdh67DQk9nuGm-oufIT0HRMPEnwePw2HDx1Vj/pub?gid=0&single=true&output=csv'
+    encoding = 'utf-8'
+
+    """
+    Returns whether the row should be imported.
+    """
+    def is_valid_row(self, row):
+        return super().is_valid_row(row) and row['organization']
+
+    def scrape(self):
+        organizations = {}
+        seat_numbers = defaultdict(lambda: defaultdict(int))
+
+        reader = self.csv_reader(self.csv_url, delimiter=self.delimiter, header=True, encoding=self.encoding, skip_rows=self.skip_rows)
+        reader.fieldnames = [self.header_converter(field) for field in reader.fieldnames]
+        for row in reader:
+
+            try:
+                if self.is_valid_row(row):
+                    for key, corrections in self.corrections.items():
+                        if not isinstance(corrections, dict):
+                            row[key] = corrections(row[key])
+                        elif row[key] in corrections:
+                            row[key] = corrections[row[key]]
+
+                    organization_classification = 'legislature'
+
+                    organization = None
```

Comment:
`organization` gets set in both branches of the if-statement, so we don't need to set it to `None` here.

Reply:
Thanks. Fixed.